tests/e2e/color-picker.spec.ts

```
@@ -0,0 +1,407 @@
+    // Verify palette is empty
+    await expect(page.locator(".palette-empty")).toBeVisible();
+  });
+
+  test("should select color from palette", async ({ page }) => {
+    // Add two colors to palette using HEX input
+    const hexInput = page.locator('input[placeholder="#000000"]');
+
+    await hexInput.clear();
+    await hexInput.fill("#FF0000");
+    await hexInput.press("Enter");
+    await page.waitForTimeout(200);
```

Biome (2.1.2)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)